tests/check_new_syntax.py

```diff
@@ -22,7 +22,7 @@ def visit_If(self, node: ast.If) -> None:
                 new_syntax = "if " + ast.unparse(node.test).replace("<", ">=", 1)
                 errors.append(
                     f"{path}:{node.lineno}: When using if/else with sys.version_info, "
-                    f"put the code for new Python versions first, e.g. `{new_syntax}`"
+                    + f"put the code for new Python versions first, e.g. `{new_syntax}`"
```

Comment:
I agree with @JelleZijlstra: I really don't like this pyright lint against implicit string concatenation. Implicit string concatenation can lead to bugs in some situations, but it's often pretty idiomatic. The pyright lint seems like it has lots of false positives that forbid idiomatic Python.

Reply:
I can turn it off. I preferred turning everything on and getting your comments :)